Zilon.Core/Zilon.Core/Commands/Sector/MoveCommand.cs

```diff
@@ -74,19 +89,28 @@ public bool CanRepeat()
         /// </summary>
         protected override void ExecuteTacticCommand()
         {
-            var selectedNodeVm = GetSelectedNodeViewModel();
-            if (selectedNodeVm == null)
+            try
             {
-                throw new InvalidOperationException("Невозможно выполнить команду на перемещение, если не указан целевой узел.");
-            }
+                var selectedNodeVm = GetSelectedNodeViewModel();
+                if (selectedNodeVm == null)
+                {
+                    throw new InvalidOperationException("Невозможно выполнить команду на перемещение, если не указан целевой узел.");
+                }
+
+                CreatePath(selectedNodeVm);
+
+                var targetNode = selectedNodeVm.Node;
 
-            CreatePath(selectedNodeVm);
+                var currentSector = _player.SectorNode.Sector;
 
-            var targetNode = selectedNodeVm.Node;
-            var targetMap = SectorManager.CurrentSector.Map;
+                var moveIntetion = new MoveIntention(targetNode, currentSector);
+                PlayerState.TaskSource.Intent(moveIntetion);
+            }
+            catch (Exception exception)
```

Comment:
Codacy found an issue: [Add logic to this catch clause or eliminate it and rethrow the exception automatically.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=6105716)